backend/docker-compose.yaml

```diff
@@ -1,4 +1,6 @@
-redis:
-    image: redis:6
-    ports:
-      - "6379:6379"
\ No newline at end of file
+version: '3.8'
+services:
```

Comment:
It's weird that others haven't run into this sooner - @positiveimpact or @ddhanesha had you?

Reply:
@piperchester I think we have to change how this work. Maybe we can move the entire local setup to docker.

Reply:
@piperchester Can you create an issue for this?